libs/blocks/modals/modals.js

```diff
@@ -57,6 +57,11 @@ async function getModal() {
 
   dialog.addEventListener('close', (e) => {
     window.location.hash = '#';
+
+    // Do not remove element if it is FaaS.
+    if (e.target.querySelector('.faas')) {
+      return;
+    }
```

Comment:
I think this is too heavy and specific. I think re-instantiating FaaS is a perfectly acceptable outcome if someone closes the modal.

Reply:
@auniverseaway  I knew this code isn't best looking, though there were two reasons I added this:
1. Waiting for loading FaaS and re-entering all the fields again aren't so fun if someone closed and re-opened the dialog accidentally (like by pressing ESC key). 
2. Fragment doesn't load the FaaS again when the modal link clicked at second time.

Since, you think it is perfectly acceptable, I can ignore the first reason. I will try to find out why fragment w/ modal doesn't like FaaS after it was terminated once.